README.md

```diff
@@ -3,4 +3,4 @@ How to create the environment variables to clone this project and run it locally
 
 - You will need to create two .env files for your project: .env.test and .env.development
 - Into each, add PGDATABASE=, with the correct database name for that environment (see /db/setup.sql for the database names)
-- Double check that these .env files are .gitignored.
+- Double check that these .env files are .gitignored
```

Comment:
Do let users know how they should set up the .env.development and .env.test files!